app/routers/CreateUserRouter.js

```diff
@@ -0,0 +1,15 @@
+var passport = require ('passport');
+
+function isLoggedIn (req,res,next) {
+    if (req.isAuthenticated ())
+        return next ();
+    res.redirect ('/login');
+}
+
+module.exports = exports = {
+    '/createUser' : {
```

Comment:
In keeping with the name change from UserCreationController, we should probably change the route to simply be /users.